docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: "3"
```

Comment:
Even though version 3 has a higher number than 2, version 3 of the docker-compose yaml schema is focussed on docker swarm. Could you change this to 2?

Reply:
done